app/views/admin/shared/forms/_email.html.haml

```diff
@@ -1,10 +1,8 @@
-.inst-box
+.py-4.sm:grid.sm:grid-cols-3.sm:gap-4.sm:py-5.border-b.border-gray-300
   %header
-    = f.label :contact_info, 'Contact Info'
-    %p.desc
-      = t('.description', type: f.object.model_name.human.downcase)
+    = f.label :contact_info, 'Contact Email'
+    -#%p.desc
+    -#  = t('.description', type: f.object.model_name.human.downcase)
 
-  = field_set_tag do
-    .row
-      .col-sm-6
-        = f.email_field :email, required: false, maxlength: 255, class: 'form-control'
+  .sm:col-span-2.max-w-lg
+    = f.email_field :email, required: false, maxlength: 255, class: 'form-control rounded-md border-gray-400 focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm h-9'
```

Comment:
what do you think about making a helper method that would return this class string, something like `input_field_classes` that could turn it into:

` = f.email_field :email, required: false, maxlength: 255, class: input_field_classes`

Reply:
definitely, that sounds great. I'll try it as a helper method. Grouping them together under a single larger utility class wasn't working properly for some reason but I'll try it as a helper method.